hand-eye-calibration/full_hand_eye_sample.py

```diff
@@ -0,0 +1,510 @@
+"""
+Script to generate a dataset and perform hand-eye calibration using a Universal Robot UR5e robot.
+The script communicates with the robot through TCP/IP.
+
+The entire sample consist of two additional files:
+    - universal_robots_hand_eye_script.urp: Robot program script that moves between different poses.
+    - robot_communication_file.xml: communication set-up file.
+
+Running the sample requires that you have universal_robots_hand_eye_script.urp on your UR5e robot,
+and robot_communication_file.xml in the same repo as this sample. Each robot pose
+must be modified to your scene. This is done in universal_robots_hand_eye_script.urp on the robot.
+
+Further explanation of this sample is found in our knowledge base:
+https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/103481384/
+"""
+import argparse
+from pathlib import Path
+import time
+import datetime
+
+import cv2
+import numpy as np
+from scipy.spatial.transform import Rotation as R
+import zivid
+import rtde.rtde as rtde
+import rtde.rtde_config as rtde_config
+
+
+def _options():
+    """Function for taking in arguments from user
+
+    Returns:
+        Arguments from user
+
+    """
+    parser = argparse.ArgumentParser(
+        description="full_hand_eye.py [-h] [--eih] [--eth] [--ip <IP address>]",
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
```

Comment:
You are not supposed to write out the brackets yourself. They are auto-generated by argparse. If you do "-h" now you'll see that it's wrong.

Do something like this?:
```
import argparse

parser = argparse.ArgumentParser(
    description='Perform Hand-Eye calibration')
mode_group = parser.add_mutually_exclusive_group(required=True)
mode_group.add_argument('--eih', "--eye-in-hand",  action='store_true',
                        help="eye-in-hand calibration")
mode_group.add_argument('--eth', "--eye-to-hand", action='store_true',
                        help="eye-to-hand calibration")
parser.add_argument("--ip", required=True,
                        help="IP address to robot")

print(parser.parse_args())
```

Reply:
can simply do this to include docstring in description,

parser = argparse.ArgumentParser(description=__doc__)

Reply:
Nice